### DISCUSSION TITLE
There is a Spanish translation?

### DISCUSSION THREAD

**Discussion by hiperiondev** [diasurgical/DevilutionX#1229] [Q&A] [answered]
I'll see an Italian translation (po file)

There is an Spanish translation?

And how is the precess for compile with the po file?

**Comment by AJenbo** [comment]
Translations are not integrated yet, but if you like you can start work on a Spanish translation.

I'm converting this to a discussion as it's more of a user question then an issue.

**Comment by hiperiondev** [comment]
Thanks for your response.

If I have a po file with translation is possible to compile with it?

**Reply by AJenbo** [reply]
PO files are not compiled by dynamically loaded depending on your selected language. And no we do not support this yet.

**Reply by hiperiondev** [reply]
There is a po file for start translation?

**Reply by AJenbo** [reply]
https://github.com/diasurgical/devilutionX/files/6030229/devilutionx-ru.zip just swithc thw lang, you can use poedit to edit.

**Comment by hiperiondev** [comment, marked as answer]
This is the Spanish translation: https://mega.nz/file/0vxmARib#lkiW2s_TUcxNZMDH63kvSIHC1_MB4q6M6FfoIp5Lhtk

Needs some corrections but is complete

**Reply by qndel** [reply]
@hiperiondev can you pack it with zip and upload directly on github? No need to upload on 3rd party websites if we can avoid it + it doesn't even let me download

**Reply by AJenbo** [reply]
https://github.com/diasurgical/devilutionX/issues/66#issuecomment-805718928

**Reply by AJenbo** [reply]
the translation has now been added

**Comment by Mad-Soft** [comment]
I think these small corrections could be of interest in the Spanish translation.
Based on the official translation of Diablo 2.

- Club = Porra
- Town Portal = Portal a la ciudad
- Portal to = Portal a
#: Source/cursor.cpp:231 Esto es así porque dice quien ha creado el portal, ya que en un multijugador pueden haber varios portales y así    # saber quien lo ha creado
msgid "from {:s}"
msgstr "de {:s}"
- "Potion of Healing" = "Poción de curación" .... y derivados
- AC es "Armor Class" = Clase de Armadura..... extra CA contra demonios"
- msgid "Zombie" = En español es "Zombi" https://dle.rae.es/zombi
- Cain the elder = Cain el anciano

**Reply by AJenbo** [reply]
@Mad-Soft thanks, but it's a little hard for us to take changes like this. Could you instead download es.po and make the corrections there. It would be preferred if you can submit the changes as a Pull Request, but if you have trouble with that then you can upload it as a zip in a comment and we ca update the project with the file.

If you have the time it would also be really appreciated if you could work on the renaming 21% (366), most of it already has suggetsion, but they need reviews (remove the fuzzy marking, can be done in Poedit by removing the "Needs Work" lable).

![image](https://user-images.githubusercontent.com/204594/137228417-aa3a53e4-49ac-4b61-ba77-c17d069b8996.png)

**Comment by Mad-Soft** [comment]
I completed and correct the remaining strings to concordate to Diablo 2 in spanish
[es.zip](https://github.com/diasurgical/devilutionX/files/7490419/es.zip)

**Reply by Mad-Soft** [reply]
How i can repack my file in devilutionx.mpq?

**Reply by qndel** [reply]
why would you want to do this?

**Reply by AJenbo** [reply]
Nice work, 100% translated, full credits. I cleaned up the file a little and committed it via https://github.com/diasurgical/devilutionX/commit/a1e9214a89f1554e47f810de7708fe0f755a2495

**Reply by Mad-Soft** [reply]
> why would you want to do this?

Because the Spanish translation file (es.gmo) is in the mpq file and I want to use this revision in the game.

**Reply by AJenbo** [reply]
Placing the es.gmo outside the mpq should overwrite the one from the mpq. Note that es.mo also works.

**Reply by Mad-Soft** [reply]
Thanks
